### PR TITLE
Added info to tables for week 4.

### PR DESCRIPTION
**Pull request by zateutsch**
Base info for week 4 tables.

### CODE REVIEW

File: docs/30DaysOfPWA/platforms-practices/07.md [hunk 1]
```diff
@@ -6,6 +6,10 @@
...
 
+
 | Section | Description |
 | ------- | ----------- |
-| | |
\ No newline at end of file
+| **Overview** |How can you create consistent design experiences for your PWA accross different platforms? What are some of the standards for UI/UX on different platforms?|
+| **Motivation** | Maintaining a consistent native user experience is key to a good PWA. |
+| **Example** | Building an application with mobile-first responsive design. |
+| **References** | Fluent, Apple Design Standards, Mobile Design |
```
Comment (aarongustafson) [inline docs/30DaysOfPWA/platforms-practices/07.md:15]: I would circle up with @ststimac to see what she’s planning to cover.

**Review by aarongustafson — APPROVED** [review]

File: docs/30DaysOfPWA/platforms-practices/07.md (hunk 1 above)
Comment (zateutsch) [reply docs/30DaysOfPWA/platforms-practices/07.md:15]: Meeting with her today.

Comment (zateutsch) [reply docs/30DaysOfPWA/platforms-practices/07.md:15]: Also plan to get updated table info from Judah for the distribution article as well.

**Review by beth-panx — APPROVED** [review]

**Comment by beth-panx** [comment]
@zateutsch is this ready?

**Comment by zateutsch** [comment]
@beth-panx  gonna bother my authors tomorrow for updated info, if not, I'll go ahead and merge

**Review by beth-panx — APPROVED** [review]
Love it! ❤️

**Pull request merged**